.github/workflows/contracts-ecdsa.yml

```
@@ -181,12 +181,31 @@ jobs:
       - name: Install dependencies
         run: yarn install --frozen-lockfile
 
-      # TODO: Configure tenderly
+      - name: Get upstream packages versions
+        uses: keep-network/ci/actions/upstream-builds-query@v2
+        id: upstream-builds-query
+        with:
+          upstream-builds: ${{ github.event.inputs.upstream_builds }}
+          query: |
+            solidity-contracts-version = github.com/threshold-network/solidity-contracts#version
+            random-beacon-version = github.com/keep-network/keep-core/random-beacon#version
+
+      - name: Resolve latest contracts
+        run: |
+          yarn upgrade \
+            @threshold-network/solidity-contracts@${{ steps.upstream-builds-query.outputs.solidity-contracts-version }} \
+            @keep-network/random-beacon@${{ steps.upstream-builds-query.outputs.random-beacon-version }} \
+            @keep-network/sortition-pools
 
+      - name: Configure tenderly
+        env:
+          TENDERLY_TOKEN: ${{ secrets.TENDERLY_TOKEN }}
+        run: ./config_tenderly.sh
```

Maybe instead of adding the script file to the repo we could execute what's needed here?
```suggestion
        run: |
            mkdir $HOME/.tenderly && touch $HOME/.tenderly/config.yaml
            echo access_key: ${TENDERLY_TOKEN} > $HOME/.tenderly/config.yaml
```

---

Hmm, I haven't figured a way to correctly escape the `:` character... When `: ` is used, whatever is before is treated as a property and causes the YAML schema to complain...

---

Let's not block this PR on that!

---

I created a proposal in https://github.com/keep-network/keep-core/pull/3141